internal/model/manifest.go

```diff
@@ -140,7 +143,7 @@ func (m1 Manifest) Equal(m2 Manifest) bool {
 	dockerignoresMatch := reflect.DeepEqual(m1.dockerignores, m2.dockerignores)
 	buildArgsMatch := reflect.DeepEqual(m1.StaticBuildArgs, m2.StaticBuildArgs)
 	cachePathsMatch := stringSlicesEqual(m1.cachePaths, m2.cachePaths)
-	dockerComposeEqual := m1.DcYAMLPath == m2.DcYAMLPath
+	dockerComposeEqual := m1.DcYAMLPath == m2.DcYAMLPath && bytes.Equal(m1.DcYAMLRaw, m2.DcYAMLRaw) && bytes.Equal(m1.DfRaw, m2.DfRaw)
```

Comment:
can you add a test for this case in `manifest_test`?

Reply:
done!